app/views/layouts/_breadcrumb.html.haml

```diff
@@ -58,5 +58,29 @@
     - else
       %li.active= @organization.abbreviation
 
+  - elsif @edit
```

Comment:
I wonder if there is a more general way to tell the user what part of the Q&A controller they are in. maybe look in params to see what controller, action and id is being requested.